Source/dom/layers/__tests__/Text.test.js

```diff
@@ -67,13 +67,14 @@ test('should change the line spacing behavior', () => {
   })
 
   // default to constant baseline
-  expect(text.lineSpacing).toBe(Text.LineSpacing.natural)
+  expect(text.lineSpacing).toBe(Text.LineSpacing.system)
```

Comment:
The `// default to constant baseline` comment needs updating, or perhaps better removal. Seems like it's not adding much (besides being factually wrong)

Reply:
👍

Reply:
updated the comment to hopefully explain better